react/assets/icons/photos.svg

```diff
@@ -1 +1 @@
-<svg role="img" xmlns="http://www.w3.org/2000/svg" width="48px" height="48px" viewBox="0 0 24 24" aria-labelledby="imageIconTitle" stroke="#2329D6" stroke-width="1" stroke-linecap="square" stroke-linejoin="miter" fill="none" color="#2329D6"> <title id="imageIconTitle">Image</title> <rect width="18" height="18" x="3" y="3"/> <path stroke-linecap="round" d="M3 14l4-4 11 11"/> <circle cx="13.5" cy="7.5" r="2.5"/> <path stroke-linecap="round" d="M13.5 16.5L21 9"/> </svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="48" height="48" viewBox="0 0 24 24" aria-labelledby="imageIconTitle" stroke="#2329D6" stroke-linecap="square" fill="none" color="#2329D6"><path d="M3 3h18v18H3z"/><path stroke-linecap="round" d="M3 14l4-4 11 11"/><circle cx="13.5" cy="7.5" r="2.5"/><path stroke-linecap="round" d="M13.5 16.5L21 9"/></svg>
```

Comment:
Revert this and update svgo according to:
https://github.com/ripe-tech/ripe-components-vue/blob/master/package.json#L48
https://github.com/ripe-tech/ripe-components-vue/blob/master/package.json#L77
https://github.com/ripe-tech/ripe-components-vue/blob/master/svgo.config.js